docs/jwst/introduction.rst

```diff
@@ -110,55 +110,30 @@ step by using the '-h' (help) argument to strun:
     $ strun dq_init.cfg -h
     $ strun jwst.pipeline.Detector1Pipeline -h
 
-If you want to consistently override the default values of certain arguments and
-don't want to specify them on the command line every time you execute ``strun``,
-you can specify them in the configuration (.cfg or .asdf) file for the pipeline
-or the individual step. For example, to always run ``Detector1Pipeline`` using
-the override in the previous example, you could edit your
-``calwebb_detector1.cfg`` file to contain the following: ::
-
- name = "Detector1Pipeline"
- class = "jwst.pipeline.Detector1Pipeline"
-
-    [steps]
-      [[dark_current]]
-        override_dark = 'my_dark.fits'
-
-Note that simply removing the entry for a step from a pipeline cfg file will
```

Comment:
I can't immediately tell if this info (about removing the entry for a step from a cfg file) is replicated somewhere else? Or has it been completely removed? This has been an issue for a lot of folks in the past, in that they think by not having any info for a step in the cfg will cause it to be skipped.

Reply:
Removed in that some of this is replicated later on, but the specific bit about the note is gone. I will recover this.